tensorrt_llm/_torch/memory_buffers_util.py

```
@@ -0,0 +1,63 @@
+        def select_buffer_with_more_elements(
+            pinned_buffer: Optional[torch.Tensor],
+            runtime_buffer: Optional[torch.Tensor]
+        ) -> tuple[torch.Tensor, bool]:
+            if pinned_buffer is None:
+                return runtime_buffer
+            if runtime_buffer is None:
+                return pinned_buffer
+            return runtime_buffer if runtime_buffer[0].numel(
+            ) > pinned_buffer[0].numel() else pinned_buffer
+
+        def view_to(buffer: torch.Tensor, dtype: torch.dtype,
```

Ruff (0.12.2)

Undefined name `torch`

(F821)